Assets/MRTK/Core/Inspectors/Setup/MixedRealityProjectConfiguratorWindow.cs

```diff
@@ -560,6 +560,14 @@ private void RenderImportTMP()
             });
         }
 
+#if UNITY_2019_3_OR_NEWER
+        [MenuItem("Mixed Reality/Toolkit/Utilities/Import Examples from Package")]
```

Comment:
Do we want to explicitly mention UPM here and say something like "Import Examples from Package (UPM Only)" or "Import Examples from UPM Package" so that people seeing this greyed out do not get confused?

Reply:
We can. The reason I did not is because the Feature Tool makes no reference to UPM

Reply:
Updated the menu text:

![image](https://user-images.githubusercontent.com/13281406/118045250-836fb080-b32c-11eb-982e-362ed56eb484.png)